modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -779,29 +779,25 @@ def set_modality(appointment)
       end
 
       def log_modality_failure(appointment)
-        Rails.logger.info("VAOS appointment id #{appointment[:id]} modality cannot be determined.")
-        if Random.new.rand(10).zero?
-          Rails.logger.info(
-            "VAOS appointment id #{appointment[:id]} modality details.",
-            {
-              service_type: appointment[:service_type],
-              service_category_text: appointment.dig(:service_category, 0, :text),
-              kind: appointment[:kind],
-              atlas: appointment.dig(:telehealth, :atlas),
-              vvs_kind: appointment.dig(:telehealth, :vvs_kind),
-              gfe: appointment.dig(:extension, :patient_has_mobile_gfe)
-            }.to_json
-          )
-        end
+        Rails.logger.warn("VAOS appointment id #{appointment[:id]} modality cannot be determined", {
```

Comment:
Setting this log level to warn and removed sampling since we are fairly confident we addressed most, if not all, of the failures.